uk.co.compucorp.civicrm.hrcore/gulp/utils.js

```diff
@@ -320,6 +324,10 @@ function spawnTaskForExtension (taskName, taskFn, extension) {
     return taskFn(cb);
   });
 
+  taskExecution = gulp.series([taskName]);
```

Comment:
you are saying that this "executes" the task, but if that's the case then this can't be right. The whole purpose of this function was not to execute the task, but just to _define_ it.

It is a programmatic way of doing this

```js
gulp.task('my-task', function () {
  // implementation of task
});
```
this simply defines the task and nothing else, it does not execute it

Reply:
I am not sure.

```js
// task definition
gulp.task('my-task', function () {
  // implementation of task
});

// task execution
gulp.series(['my-task']);
// or
gulp.parallel(['my-task']);
```